leetcode/medium/227._Basic_Calculator_II.py

```diff
@@ -0,0 +1,51 @@
+# Link to problem: https://leetcode.com/problems/basic-calculator-ii/description/
+# Loom explaining solution you wrote (5m): 
+# Solution:
+# Description
+# Key ideas
+# To step process, process */ first, +- last.
+# - can be treated as + (-val), +- can be mapped as sum
+# When processing */, use a value stack
+# Difficulty: Medium
+# Code
+
+
+def token_of(s):
```

Comment:
Please add typing to this helper function 

```
def tokenize(s: str) -> List[str]:
```

This function is usually called tokenize.

Reply:
understood.